.github/actions/setup/action.yml

```diff
@@ -37,16 +36,15 @@ runs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Install Vorta
+    - name: Install Nox and pre-commit
       shell: bash
-      run: |
-        pip install -e .
-        pip install -r requirements.d/dev.txt
+      run: pip install nox pre-commit
```

Comment:
Since this action is used by lint also, what about adding a `nox` input?

Reply:
Added.